### PR TITLE
chatapi: docker Implementation of node proxy service for openAI API (fixes #7153)

### LINKED ISSUE #7153: Experiment: Connecting Planet with prompting LLMs
Developing a page to allow access to LLMs from within planet

- Separate(Hidden) route that can be easily rolled back
- Integrate with GPT4 over API first and potentially later Google Bard & Vicuna 13b
- Form with 2 fields to allow that allows submitting prompt
- Display the output of the API call/prompt

### PR DESCRIPTION
**Pull request by Mutugiii**
Creating a node.js proxy service for the GPT/LLM API calls

Fixes #7153

### CODE REVIEW

File: docker/chat-api/arm-Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial as builder
+LABEL maintainer="sahil@ole.org,mappuji@ole.org"
+
+COPY docker/db-init/crosscompile_db-init.sh .
```
Comment (lmmrssa) [inline docker/chat-api/arm-Dockerfile:4]: is this db init folder?

File: docker/chat-api/arm-Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    apk add --no-cache bash curl git jq ca-certificates;\
+    mkdir -p /usr/local/lib/node_modules; \
+    ln -s /usr/local/lib/node_modules/add-cors-to-couchdb/bin.js /usr/local/bin/add-cors-to-couchdb
+
+COPY --from=builder /usr/lib/node_modules /usr/local/lib/node_modules
+
+WORKDIR /root
+
+COPY ./docker/db-init/docker-entrypoint.sh /root/docker-entrypoint.sh
+COPY ./couchdb-setup.sh /root/couchdb-setup.sh
```
Comment (lmmrssa) [inline docker/chat-api/arm-Dockerfile:21]: not sure if this is required as planet already run this init

File: docker/chat-api/arm-Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ADD ./design /root/design
```
Comment (lmmrssa) [inline docker/chat-api/arm-Dockerfile:22]: this is also not needed

File: docker/planet/arm64-Dockerfile [hunk 4]
```diff
@@ -13,13 +13,13 @@ RUN apk add --no-cache \
     jq \
       openssl
 
-COPY docker/planet/default.conf /etc/nginx/conf.d/
+COPY ./docker/planet/default.conf /etc/nginx/conf.d/
```
Comment (lmmrssa) [inline docker/planet/arm64-Dockerfile:16]: there were working fine. any reason for changing them?

Comment (Mutugiii) [reply docker/planet/arm64-Dockerfile:16]: suggestion from @dogi to be standard across the dockerfiles

**Review by lmmrssa — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**